.php_cs.dist

```diff
@@ -9,33 +9,33 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 HEADER;
 
-$rules = [
```

Comment:
Do we need to change how we build the rules on master?

Reply:
Rules will differs it's sure because of the dedicated rules to PHP7.1+ and PHPUnit6+ (and the updated version of PHP-CS-Fixer itself)
And I tend to follow the trend to reduce use of phpdoc with PHP7+ new features that are more explicit about arguments/return